libs/sdk-ui-kit/styles/scss/form.scss

```diff
@@ -345,13 +345,13 @@ textarea {
             width: 12px;
             height: 12px;
             border: 1px solid $gd-input-text-hover-border;
-            transition: all 0.2s;
+            // transition: all 0.2s; TODO: disable transition only during the scroll
             background-color: $gd-color-text-light;
         }
 
         &::after {
             transform: scale(0);
-            transition: transform 0.2s;
+            // transition: transform 0.2s; TODO: disable transition only during the scroll
```

Comment:
Is this change save? It looks like all forms input-radio-label,
.input-checkbox-label are now without transition